src/components/MainSite.astro

```diff
@@ -11,7 +12,16 @@ export interface Props {
 const { websiteAssets, id } = Astro.props;
 ---
 
-<div class="max-w-8xl m-auto">
+<div class="w-full m-auto">
+	<div class="absolute w-full h-full">
+		<div
+			class="middle hidden dark:block absolute animate-wigglemiddle <md:hidden left-[50%] h-full min-h-full cursor-ew-resize z-20">
+			<div
+				class="fixed transform -translate-x-2/4 w-10 h-10 border-1 bg-white border-[#FFAD8A] top-11/36 grid grid-cols-[1fr_1fr] rounded-1/2 items-center">
+				<img src={Short_Logo} class="ml-1/2 pointer-events-none" />
+			</div>
+		</div>
+	</div>
```

Comment:
Apparently the issue is with this div, removing it makes the buttons work again.

Reply:
yep already removed, I've put it there by mistake